File: src/main/java/com/poixpixelcustom/util/BukkitTools.java
package com.poixpixelcustom.util;

import com.poixpixelcustom.PoixpixelCustom;
import org.bukkit.Server;

public class BukkitTools {
    private static PoixpixelCustom plugin = null;
    private static Server server = null;

    public static void initialize(PoixpixelCustom plugin) {
        BukkitTools.plugin = plugin;
        BukkitTools.server = plugin.getServer();
    }

}

File: src/main/java/com/poixpixelcustom/util/BukkitTools.kt
package com.poixpixelcustom.util

import com.poixpixelcustom.PoixpixelCustom
import org.bukkit.Server

object BukkitTools {
    private var plugin: PoixpixelCustom? = null
    private var server: Server? = null
    fun initialize(plugin: PoixpixelCustom) {
        BukkitTools.plugin = plugin
        server = plugin.server
    }

}